src/web/mjs/connectors/ComicBrise.mjs

```diff
@@ -0,0 +1,48 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ComicBrise extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'comicbrise';
+        super.label = 'comicBrise';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://comic-brise.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.post-title');
+        let id = uri.pathname;
+        let title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const msg = 'This website does not support mangas/chapters, please copy and paste the links containing the chapters directly from your browser into HakuNeko.';
+        throw new Error(msg);
+    }
+
+    async _getChapters(manga) {
+        const uri = new URL(manga.id, this.url);
+        const request = new Request(uri, this.requestOptions);
+		const dom = await this.fetchDOM(request);
+        const chaptersNames = dom.querySelectorAll('.modal.modal-chapter .modal-body .primary-title');
+        const chaptersLink = dom.querySelectorAll('.modal.modal-chapter .modal-body .banner-trial a');
+        const chaptersFree = dom.querySelectorAll('.modal.modal-chapter .modal-body .banner-trial source');
```

Comment:
Consider to query against `.modal.modal-chapter .modal-body` and then map the list by query each element itself,
e.g.
```javascript
const id = element.querySelector('.banner-trial a').pathname;
const title = element.querySelector('.primary-title').textContent.trim();
// ...
```

Reply:
May utilize the `filter` function to exclude non-free titles